components/directory-server-manager/org.wso2.carbon.directory.server.manager/src/main/java/org/wso2/carbon/directory/server/manager/DirectoryServerApplicationMgtListener.java

```diff
@@ -95,7 +96,7 @@ public boolean doPreUpdateApplication(ServiceProvider serviceProvider, String te
             for (InboundAuthenticationRequestConfig authConfig
                     : inboundAuthenticationConfig.getInboundAuthenticationRequestConfigs()) {
 
-                if (StringUtils.equals(authConfig.getInboundAuthType(), "kerberos")) {
+                if (StringUtils.equals(authConfig.getInboundAuthType(), KERBEROS)) {
```

Comment:
Since KERBEROS is a constant, we can directly use KERBEROS.equals(authConfig.getInboundAuthType()) here